gear-programs/erc20-relay/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "erc20-relay"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+erc20-relay-app.workspace = true
+
+[build-dependencies]
+erc20-relay-app.workspace = true
+sails-rs = { git = "https://github.com/gear-tech/sails.git", rev = "aab08733ee9ccdb809dc9b29c57dd411b2b917b1", features = ["wasm-builder"] }
+sails-idl-gen = { git = "https://github.com/gear-tech/sails.git", rev = "aab08733ee9ccdb809dc9b29c57dd411b2b917b1" }

```

Comment:
If you plan to merge it without waiting for `sails` release then please open an issue and link it here

Reply:
And can it be moved to sworkspace deps to not forget to update it everywhere in future?